aci/utils.go

```diff
@@ -469,3 +469,15 @@ func setModelAttributes(modelAttr interface{}, modelAttrMap map[string]string) e
 	}
 	return nil
 }
+
+func toggleBetweenYesEnabledNoDisabled(hub_networking string) string {
```

Comment:
Do we want separate toggle for each, or do we want a generic bigger function? Because we should consider a function name change.

Also for readability did you consider switch statement or map lookup? They likely will have speed disadvantages over if-else but not sure if that would impact us too much.

Reply:
Let me create a more general function, so that we can expand it when the need arises